header/src/main/filtered-resources/WEB-INF/jsp/index.jsp

```diff
@@ -215,7 +240,7 @@ if(sec_roles != null) {
         <c:choose>
             <c:when test='<%= anonymous == false %>'>
         <p class="logged">
-            <a href="${ldapadminPublicContextPath}/account/userdetails"><%=request.getHeader("sec-username") %></a><span class="light"> | </span><a href="/j_spring_security_logout"><fmt:message key="logout"/></a>
+            <a href="<%=ldapadm %>/account/userdetails"><%=request.getHeader("sec-username") %></a><span class="light"> | </span><a href="/j_spring_security_logout"><fmt:message key="logout"/></a>
```

Comment:
If fixed into the .properties file, there is nothing more to do here, right ? (or else I did not get the comment)